app/src/main/res/values/strings.xml

```diff
@@ -250,18 +244,24 @@
     <!-- This is the the button description on the device security dialog box which routes to system security settings. -->
     <string name="set_up_now">Set Up Now</string>
 
-    <!-- This header appears above the Privacy page of the mozilla-lockbox website. -->
+    <!-- This header appears above the Privacy page of the application website. -->
     <string name="privacy">Privacy</string>
 
-    <!-- This is the prompt when autofill is triggered, and Firefox Lockbox is locked. -->
-    <string name="autofill_authenticate_cta">Unlock Firefox Lockbox</string>
+    <!-- This is the prompt when autofill is triggered and the application is locked. %1$S will be replaced with application name -->
+    <string name="autofill_authenticate_cta">Unlock %1$S</string>
     <!-- This is the error message when autofill is triggered but no domain is detected. This is unlikely. -->
     <string name="autofill_error_no_hostname">Unexpected hostname format</string>
-    <!-- This is the prompt when autofill is triggered, Lockbox is unlocked, and no passwords are detected. -->
-    <string name="autofill_search_cta">Search Firefox Lockbox</string>
-    <!-- This is the error message when autofill is triggered and Lockbox is in an error state. %s refers to the localized error message -->
-    <string name="autofill_error_toast">Firefox Lockbox encountered an error: %s</string>
+    <!-- This is the prompt when autofill is triggered, application is unlocked, and no passwords are detected. %1$S will be replaced with application name. -->
+    <string name="autofill_search_cta">Search %1$S</string>
+    <!-- This is the error message when autofill is triggered and the application is in an error state. %s refers to the localized error message -->
```

Comment:
Please also explain in this comment to what the placeholder will be replaced with :)

Reply:
>Please also explain in this comment to what the placeholder will be replaced with :)


As I understand it, the error message will be returned from the system so it could be any number of things and would not be translated.